.github/workflows/ci.yml

```diff
@@ -143,6 +144,15 @@ jobs:
           cd build
           mkdir dist
           mv *.3dsx dist/
+      
+      - name: Package (Xbox One)
```

Comment:
This can be removed: you added install instructions to the cmake script.

Reply:
cpack contains not only msix and deps but also exe which is unnecessary
if it's fine then i'll change it

Reply:
Installing the raw binary does not make sense for more platforms.
e.g. on n3ds, the elf is also useless (ignoring debugging)

We want to use CPack here, and not the manual packaging to avoid creating a continuous release with all xbos files separate: they should be grouped.

So I'd be fine with doing something as:
```cmake
if(NOT (NINTENDO_3DS OR WINDOWS_STORE))
  install(TARGETS isle ...)
endif()
```

Reply:
The `Xbox One` archive looks empty on ci.